Android/Shared/src/main/java/com/bentley/sample/shared/SampleITMApplication.kt

```diff
@@ -80,27 +77,27 @@ open class SampleITMApplication(context: Context, attachWebViewLogger: Boolean,
             }
         }
 
-        coMessenger.registerMessageHandler("firstRenderStarted") {
+        coMessenger.registerMessageHandler<Unit>("firstRenderStarted") {
             logger.log(ITMLogger.Severity.Debug, "Received firstRenderStarted")
         }
-        coMessenger.registerMessageHandler("firstRenderFinished") {
+        coMessenger.registerMessageHandler<Unit>("firstRenderFinished") {
             logger.log(ITMLogger.Severity.Debug, "Received firstRenderFinished")
         }
     }
 
     /**
-     * Checks for values in configData with a prefix of "ITMSAMPLE_ACTION_" and constructs a [JsonObject]
+     * Checks for values in configData with a prefix of "ITMSAMPLE_ACTION_" and constructs a [JSONObject]
      * using everything after ITMSAMPLE_ACTION_ as the key and the values from configData.
-     * @return A [JsonObject] containing all the ITMSAMPLE_ACTION_ prefixed values from configData.
+     * @return A [JSONObject] containing all the ITMSAMPLE_ACTION_ prefixed values from configData.
      */
-    protected open fun getActionsFromConfigData(): JsonObject {
-        val actions = JsonObject()
+    protected open fun getActionsFromConfigData(): JSONObject {
+        val actions = JSONObject()
         configData?.let { configData ->
-            for (key in configData.names()) {
-                configData.getOptionalString(key)?.let { value ->
-                    val shortKey = key.removePrefix("ITMSAMPLE_ACTION_")
-                    if (shortKey.length < key.length) {
-                        actions[shortKey] = value
+            configData.toMap().forEach { entry ->
```

Comment:
These two lines can be combined into one as the `?.let` is unnecessary and can be replaced by `configData?.toMap...`

Reply:
Fixed.